suite-native/module-add-accounts/src/screens/SelectAccountTypeScreen.tsx

```diff
@@ -33,8 +31,8 @@ const bulletsForKeyPath = (keyPath: TxKeyPath) => (
         id={keyPath}
         values={{
             li: chunks =>
-                chunks.map(row => (
-                    <Box flexDirection="row">
+                chunks.map((row, index) => (
+                    <Box key={`${keyPath}-${index}`} flexDirection="row">
```

Comment:
I believe the `row` should be enough as a key. In case that the `row` is undefined, the map should not return a ReactNode at all.

Reply:
If there is the same text for multiple account types, it will end up having the same key for multiple bullets. This should make it uniqe for all various account types

Reply:
09fa3a9df3

Reply:
Yes, it is possible that there will be same row strings across all the account types. But they won't be rendered in the same parent element, so I believe it is not a problem. It would be troublesome, If you would render them right next to each other.